docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -96,9 +98,121 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Creating systemd unit file for a pod with `--new` flag
```

Comment:
```suggestion
Creating systemd unit file for a container with `--new` flag
```

Pods are not yet supported.

Reply:
Can you add a note to `--new` in the OPTIONS section that pods are not yet supported?